go-ssf-set/README.md

```diff
@@ -0,0 +1,454 @@
+# Go SSF SET Library
+
+A comprehensive Go library for building, signing, parsing, and validating Security Event Tokens (SETs) according to the Shared Signals Framework (SSF) specification.
```

Comment:
SETs are actually independent of SSF, so we should decide at a high level whether we want to let this be used in non-SSF applications (which would be fine), or restrict it to SSF. We could say "...and validating SETs for use in SSF" if needed, but saying "according to SSF" is probably incorrect.